docs/simbad/query_tap.rst

```diff
@@ -0,0 +1,322 @@
+`~astroquery.simbad.SimbadClass.query_tap` (for Table Access Protocol) is the one
+query to rule them all. It allows to interrogate all the information in SIMBAD with the
```

Comment:
```suggestion
query to rule them all. It allows one to access all the information in SIMBAD with the
```
Minor English, but "Allow to <verb>" isn't an accepted construction.  Also, "interrogate" carries connotations that mean it's not so commonly used; it's technically acceptable, but I'd recommend "access" or "query" instead.